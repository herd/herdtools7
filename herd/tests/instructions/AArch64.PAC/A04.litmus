AArch64 A04
Variant=pac,fpac,const-pac-field

{ 0:x0=pacda(x,0); 0:x1=0; int x=0; 0:x2=42 }

P0                 ;
  cas x1, x2, [x0] ;
forall
  ( Fault(P0, MMU:Translation) )
