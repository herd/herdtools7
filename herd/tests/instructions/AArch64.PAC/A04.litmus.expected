Test A04 Required
States 1
 Fault(P0,pacda(x, 0x0, 0),MMU:Translation);
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (fault(P0,MMU:Translation))
Observation A04 Always 1 0
Hash=889fcb9c83865acded62c17df51f4415

